tests/test_other.py

```diff
@@ -243,26 +243,6 @@ def test_emcc_basics(self, compiler, suffix):
     self.assertContained(["error: use of undeclared identifier 'cheez", "error: unknown type name 'cheez'"], stderr)
     self.assertContained('errors generated.', stderr.splitlines()[-2])
 
-  @parameterized({
-    'c': [EMCC, '.c'],
-    'cxx': [EMXX, '.cpp']})
-  def test_emcc_2(self, compiler, suffix):
-    # emcc src.cpp -c    and   emcc src.cpp -o src.[o|bc] ==> should give a .bc file
```

Comment:
do we have other tests that check `-c` emitting?

Reply:
I'll revert this.. although I'm not sure this test is testing anything useful these days.    

Its probably worth revisiting because I think it used to test without `-c`.. not it always uses `-c` .. and its really not clear what its testing.